package.json

```diff
@@ -1,7 +1,8 @@
 {
   "version": "1.0.7",
   "devDependencies": {
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6"
+    "@types/node": "^18.7.2",
+    "@vtex/prettier-config": "^1.0.0",
+    "typescript": "^4.7.4"
```

Comment:
Não tenho certeza se essa versão funciona 🤔. Pelo que vi [nessa thread](https://vtex.slack.com/archives/C01PK5NGCUC/p1641931806021800), o BuilderHub força a versão para a [mesma versão dele](https://github.com/vtex/builder-hub/blob/master/node/package.json#L104)

Reply:
Não sei quais seriam os impactos de deixarmos nessa versão (se ele força em background e a gente não perceberia, ou se ele não deixaria publicar a app)

Reply:
ótimo ponto, @rodrigoap2 ! Deixarei essa versão fixa seguindo a usada no BuilderHub.